plugins/woocommerce/tests/e2e-pw/fixtures/install-wc.setup.js

```diff
@@ -0,0 +1,144 @@
+	if ( activatedWcVersion ) {
+		console.log(
+			`WooCommerce is activated. Version: ${ activatedWcVersion }`
+		);
+	} else {
+		console.log( 'WooCommerce is not activated.' );
+	}
+
+	const wcVersion = process.env.WC_VERSION || 'latest';
+	let resolvedVersion = '';
+
+	// Install WC
+	if ( wcVersion === 'latest' ) {
+		const latestResponse = await wcbtApi.fetch(
+			'/wp-json/wc-admin-test-helper/live-branches/install/latest/v1',
+			{
+				method: 'POST',
+				data: { include_pre_releases: true },
+			}
+		);
+
+		if ( ! latestResponse.ok() ) {
+			throw new Error(
+				`Failed to install latest WC: ${ latestResponse.status() } ${ await latestResponse.text() }`
+			);
+		}
+
+		resolvedVersion = ( await latestResponse.json() )?.version || '';
+
+		setup.skip(
+			resolvedVersion === activatedWcVersion,
+			'Skip installing WC: The latest version is already installed and activated.'
+		);
+
+		if ( ! resolvedVersion ) {
+			console.error( 'Error: latestResponse.version is undefined.' );
+		} else {
+			console.log( `Latest version installed: ${ resolvedVersion }` );
+		}
+		if ( activatedWcVersion ) {
+			await deactivateWooCommerce( wcbtApi );
+		}
+	} else {
+		setup.skip(
+			wcVersion === activatedWcVersion,
+			'Skip installing WC: Specified version is already installed and activated.'
+		);
+		if ( activatedWcVersion ) {
+			await deactivateWooCommerce( wcbtApi );
+		}
+		try {
+			const downloadUrl = `https://github.com/woocommerce/woocommerce/releases/download/${ wcVersion }/woocommerce.zip`;
+			const installResponse = await wcbtApi.fetch(
+				'/wp-json/wc-admin-test-helper/live-branches/install/v1',
+				{
+					method: 'POST',
+					data: {
+						pr_name: wcVersion,
+						download_url: downloadUrl,
+						version: wcVersion,
+					},
+				}
+			);
+
+			if ( ! installResponse.ok() ) {
+				throw new Error(
+					`Failed to install WC ${ wcVersion }: ${ installResponse.status() } ${ await installResponse.text() }`
+				);
+			}
+
+			resolvedVersion = wcVersion;
+			console.log( `WooCommerce ${ wcVersion } installed.` );
+		} catch ( err ) {
+			console.error( `Error installing WC version ${ wcVersion }:`, err );
+		}
+	}
+
+	// Activate WC
+	if ( resolvedVersion ) {
+		try {
+			const activationResponse = await wcbtApi.fetch(
+				'/wp-json/wc-admin-test-helper/live-branches/activate/v1',
+				{
+					method: 'POST',
+					data: {
+						version: resolvedVersion,
+					},
+				}
+			);
+
+			if ( ! activationResponse.ok() ) {
+				throw new Error(
+					`Failed to activate WC ${ resolvedVersion }: ${ activationResponse.status() } ${ await activationResponse.text() }`
+				);
+			}
+
+			console.log( `WooCommerce ${ resolvedVersion } activated.` );
+		} catch ( err ) {
+			console.error(
+				`Error activating WC version ${ resolvedVersion }:`,
+				err
+			);
+		}
+	} else {
+		console.error(
+			'Error: resolvedVersion is undefined. Skipping activation.'
+		);
+	}
+
+	console.log( 'Installing with WC Beta Tester is finished.' );
```

Comment:
Can we also add a check that the plugin version is actually installed and active, similar to the initial `wp-json/wp/v2/plugins` check.